test/common/services/kibana_server/ui_settings.js

```diff
@@ -1,46 +1,47 @@
 import { get } from 'lodash';
+import toPath from 'lodash/internal/toPath';
+
+import { SavedObjectsClient } from '../../../../src/server/saved_objects';
+import kibanaCoreMappings from '../../../../src/core_plugins/kibana/mappings';
+
+function createCallCluster(es) {
+  return function callCluster(method, params) {
+    const path = toPath(method);
+    const contextPath = path.slice(0, -1);
+
+    const action = get(es, path);
+    const context = contextPath.length ? get(es, contextPath) : es;
+
+    return action.call(context, params);
+  };
+}
 
 export class KibanaServerUiSettings {
   constructor(log, es, kibanaVersion) {
-    this.es = es;
-    this.log = log;
-    this.kibanaVersion = kibanaVersion;
+    this._log = log;
+    this._kibanaVersion = kibanaVersion;
+    this._savedObjectsClient = new SavedObjectsClient(
+      '.kibana',
```

Comment:
Not in the functional tests, it's not configurable there since the server is always running with the same config and the archives are setup to use `.kibana`